lib/doc/worksheet.js

```diff
@@ -782,8 +783,32 @@ class Worksheet {
     this._media = value.media.map(medium => new Image(this, medium));
     this.sheetProtection = value.sheetProtection;
     this.tables = value.tables.reduce((tables, table) => {
-      const t = new Table();
-      t.model = table;
+      /**
+       * Once we have the table information and the sheet we can add the rows related to it.
+       * We parse the start and end via the reference and then we iterate over all the
+       * cells the sheet has to see if they are inside the table to add their values to its rows.
+       */
+      const [,startColumnStr, startRowStr, endColumnStr, endRowStr] = /(\D+)(\d+):(\D+)(\d+)/i.exec(table.tableRef);
+      const startColumn = startColumnStr.toUpperCase().charCodeAt(0) - 64;
+      const endColumn = endColumnStr.toUpperCase().charCodeAt(0) - 64;
```

Comment:
This only works with columns with one character, not with `AA`.
Columns are always letters, right?

Reply:
Support for multiple letters seams be required to approval

Reply:
Yes, that's why I mentioned it. Also didn't want to spend more time if this did not have a chance to be merged as it was already solving my issues 😊

Reply:
@molant I think you can use the l2n helper from the col-cache utils file

Reply:
Use this to get the cell index, instead of doing that manual calculations, then any cell can be specified.
 
```const startIndex = worksheet.getCell(startCell).col```